### DISCUSSION TITLE
How to set table header width using column props?

### DISCUSSION THREAD

**Discussion by jeevasusej** [TanStack/table#4290] [Q&A]
Hi, 

I am using Table v8.
I want to set width in percentage for each column. But I couldn't do that and I cannot find it in the document. 
How to set the each column width?

**Comment by Schmell** [comment]
I don't believe you can do this with columnDefs
You can control columns by conditionals while building the table in jsx.
``` jsx
<Tr key={headerGroup.id}>
    {headerGroup.headers.map((header) => {
    return (
        <Th
        key={header.id}
        colSpan={header.colSpan}
        color={"gray.700"}
        // pick what column you want
        width={header.id === "points" ? "15px" : "unset"}
        >
```
Keep in mind that the other columns will probably try to fill available space
